tests/load/setup_k8s.sh

```diff
@@ -10,7 +10,7 @@ CLUSTER='autopush-locust-load-test'
 TARGET='https://updates-autopush.stage.mozaws.net'
 SCOPE='https://www.googleapis.com/auth/cloud-platform'
 REGION='us-central1'
-WORKER_COUNT=300
+WORKER_COUNT=${WORKER_COUNT:-300}
```

Comment:
The WORKER_COUNT found here has to correspond with the constant of the same name in [load.py](https://github.com/mozilla-services/autopush-rs/blob/master/tests/load/locustfiles/load.py#L47).

I also just noticed the description on that class has an [erroneous total user count in the description](https://github.com/mozilla-services/autopush-rs/blob/master/tests/load/locustfiles/load.py#L39C8-L39C82)

Reply:
I updated [load.py](https://github.com/mozilla-services/autopush-rs/pull/584/files#diff-7d05ac34f9a820145966c4583f25e689fbcb439fe6ec064f7cfa19c7cb9e8950R50) to also use the `WORKER_COUNT` environment variable, if it's specified.